packages/website/ts/blockchain.ts

```diff
@@ -161,7 +161,7 @@ export class Blockchain {
             // We catch all requests involving a users account and send it to the injectedWeb3
             // instance. All other requests go to the public hosted node.
             const provider = new Web3ProviderEngine();
-            provider.addProvider(new SignerSubprovider(injectedWeb3.currentProvider));
+            provider.addProvider(new MetamaskSubprovider(injectedWeb3.currentProvider));
```

Comment:
Wait, we only want to do this IF `provider.isMetamask` is true. Let's add that check. Website should also work with other injected signers.